jobs/service-fabrik-serviceflow-manager/spec

```diff
@@ -0,0 +1,23 @@
+---
+name: service-fabrik-serviceflow-manager
+
+packages:
+  - bosh-helpers
+  - node
+  - service-fabrik-broker
+
+templates:
+  bin/service-fabrik-serviceflow-manager_ctl.erb: bin/service-fabrik-serviceflow-manager_ctl
+  bin/job_properties.sh.erb: bin/job_properties.sh
+  config/settings.yml.erb: config/settings.yml
+  config/syslog_forwarding.conf.erb: config/syslog_forwarding.conf
+
+consumes:
+- name: broker
+  type: broker
+- name: scheduler
+  type: scheduler
+- name: deployment_hooks
```

Comment:
Is this really needed? Will it be possible to strip down `settings.yml` for this job in order to remove unnecessary properties.

Reply:
Removed